doc/api/url.md

```diff
@@ -1007,7 +1008,8 @@ the new `URL` implementation but is not part of the WHATWG URL standard.
 * `domain` {String}
 * Returns: {String}
 
-Returns the Unicode serialization of the `domain`.
+Returns the Unicode serialization of the `domain`. If `domain` is an invalid
```

Comment:
Btw, should this be `deserialization`, and mention that it is the inverse of `domainToASCII`?